airflow/contrib/operators/qubole_operator.py

```diff
@@ -117,7 +117,7 @@ class QuboleOperator(BaseOperator):
                        'extract_query', 'boundary_query', 'macros', 'name', 'parameters',
                        'dbtap_id', 'hive_table', 'db_table', 'split_column', 'note_id',
                        'db_update_keys', 'export_dir', 'partition_spec', 'qubole_conn_id',
-                       'arguments', 'user_program_arguments')
+                       'arguments', 'user_program_arguments', 'cluster_label')
```

Comment:
if user provides params.cluster_label, do we still need this line?

Reply:
Airflow will look at `template_fields` to find fields that it need to be rendered. https://github.com/lyft/incubator-airflow/blob/data-platform-upgrade/airflow/models.py#L1603

If we don't add this line, `{{ params.cluster_label }}` won't render to `default`.